group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilder.java

```diff
@@ -377,19 +375,19 @@ private Assignment newMemberAssignment(
     ) {
         MemberAssignment newMemberAssignment = newGroupAssignment.members().get(memberId);
         if (newMemberAssignment != null) {
-            return new Assignment(newMemberAssignment.targetPartitions());
+            return new Assignment(newMemberAssignment.partitions());
         } else {
             return Assignment.EMPTY;
         }
     }
 
     // private for testing
-    static MemberSubscriptionSpecImpl createMemberSubscriptionSpecImpl(
+    static MemberSubscriptionAndAssignmentImpl createMemberSubscriptionSpecImpl(
```

Comment:
Should it get renamed according to `MemberSubscriptionAndAssignmentImpl`? or it can be a constructor of `MemberSubscriptionSpecImpl`?

Reply:
Fixed it!